From 59611509b4281de86d9873b45e364719ce2f8a44 Mon Sep 17 00:00:00 2001
From: Jason Chen <jason.chen@freecale.com>
Date: Thu, 13 Mar 2008 15:23:06 +0800
Subject: [PATCH] ENGR00059726  IDMA hang issue

High IDMA requests service number will cause the IDMA channel hang.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 drivers/mxc/ipu/ipu_common.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mxc/ipu/ipu_common.c b/drivers/mxc/ipu/ipu_common.c
index 6d6eef0..00ff5bc 100644
--- a/drivers/mxc/ipu/ipu_common.c
+++ b/drivers/mxc/ipu/ipu_common.c
@@ -135,7 +135,7 @@ int ipu_probe(struct platform_device *pdev)
 	__raw_writel(0x0000C000L, IDMAC_CHA_PRI);
 
 	/* Set to max back to back burst requests */
-	__raw_writel(0x00000070L, IDMAC_CONF);
+	__raw_writel(0x00000000L, IDMAC_CONF);
 
 	register_ipu_device();
 
-- 
1.5.4.4

